docs/api.rst

```diff
@@ -4445,6 +4445,69 @@ Querying complete action history
    :statuscode 409: No user is currently logged in.
    :statuscode 500: Internal rotki error.
 
+
+Querying complete action history for debugging purposes
+=======================================================
+
+.. http:get:: /api/(version)/history/debug
+
+   .. note::
+      This endpoint can also be queried asynchronously by using ``"async_query": true``
+
+   .. note::
+      This endpoint also accepts parameters as query arguments.
+
+   Doing a GET on the history endpoint will trigger a query and processing of the history of all actions (trades, deposits, withdrawals, loans, eth transactions) within a specific time range alongside the user settings and ignored events identifiers.
```

Comment:
/s/GET/POST
```suggestion
   Doing a  POST on the PnL report debug endpoint will trigger a query and processing...
```

Also this should not just trigger the history. In fact it should not process the history, just create it and export it.